hardware/raspberrypi/bcm2711_bootloader_config.md

```diff
@@ -43,89 +18,92 @@ This section describes all the configuration items available in the bootloader.
 
 If 1 then enable UART debug output on GPIO 14 and 15. Configure the receiving debug terminal at 115200bps, 8 bits, no parity bits, 1 stop bit.
 
-Default: 0  
+Default: `0`  
 Version: All  
 
 ### WAKE_ON_GPIO
 
 If 1 then 'sudo halt' will run in a lower power mode until either GPIO3 or GLOBAL_EN are shorted to ground.
 
-Default: 1 (0 in original version of bootloader 2019-05-10)  
+Default: `1` (`0` in original version of bootloader 2019-05-10)  
 Version: All  
 
 ### POWER_OFF_ON_HALT
 
 If 1 and WAKE_ON_GPIO=0 then switch off all PMIC outputs in halt. This is lowest possible power state for halt but may cause problems with some HATs because 5V will still be on. GLOBAL_EN must be shorted to ground to boot.
 
-Default: 0  
+Default: `0`  
 Version: 2019-07-15  
 
 ### BOOT_ORDER
 The BOOT_ORDER setting allows flexible configuration for the priority of different bootmodes. It is represented as 32bit unsigned integer where each nibble represents a bootmode. The bootmodes are attempted in lowest significant nibble to highest significant nibble order.
 
-E.g. 0x21 means try SD first followed by network boot then stop. Whereas 0x2 would mean try network boot and then stop without trying to boot from the SD card.
+E.g. `0x21` means try SD first followed by network boot then stop. Whereas `0x2` would mean try network boot and then stop without trying to boot from the SD card.
```

Comment:
I think this would be easier to understand if the second sentence read as follows:

`Whereas 0x2 would mean try network boot only.` (with backticks round 0x2).